compare_usafacts_nyt_083120.Rmd

```diff
@@ -0,0 +1,427 @@
+
+**New York Times (NYT)**: The New York times keeps maintains a [data set](https://github.com/nytimes/covid-19-data) reporting daily cumulative counts of COVID-19 cases and deaths.
+
+**USAFacts**: USAFacts also maintains a [data set](https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/) reporting daily cumulative counts of COVID-19 cases and deaths.
+
+The data for each of the two sources are gathered from state and local public health agencies, however there are some differences between the two sources in how the data is reported. This [blog post](https://source.opennews.org/articles/comparison-four-major-covid-19-data-sources/) reviews some of the major differences, for example:
+
+1) **Time of update:** NYT is updated once daily around 8am EDT with case and death counts from the previous day. USAFacts is updated once daily around 12am PDT (3am EDT) for cases from the previous day. Because USAFacts is updated before NYT, USAFacts generally reports lower case and death counts than NYT for a given day.
+
+2) **Geographic exceptions:** NYT reports cases for all five boroughs of New York city as a single area ("New York City"). USAFacts reports New York City cases separately for all five counties, though also has a category for “New York City Unallocated/Probable”. NYT reports all case for Kansas City in a single group ("Kansas City"). The case counts for the four counties that include parts of Kansas City represent only the parts of the counties that are outside of Kansas City. In the USAFacts data, all Kansas City cases are allocated to Jackson County.
+
+3) **Other differences:** See the blog post linked above for more information on the differences between the data sources.
+
+
+## Section 1: Comparison of total national cases and deaths over time {.tabset}
+`r last_update_str`
+
+```{r}
+nyt_total <- dat_nyt %>% filter(state!="Puerto Rico") %>%
+  group_by(date) %>% 
+  summarise(cases=sum(cases), deaths=sum(deaths), source="NYT") 
+usafacts_total_cases <- dat_usafacts_cases %>% group_by(date) %>% summarise(cases=sum(cases)) 
+usafacts_total_deaths <- dat_usafacts_deaths %>% group_by(date) %>% summarise(deaths=sum(deaths)) 
+usafacts_total = full_join(usafacts_total_cases, usafacts_total_deaths, by="date") %>% mutate(source="USAFacts")
+```
+
+### Cases
+The following plot shows daily cumulative case counts for each of the two data sources, as well as the difference in case counts (NYT-USAFacts) and percentage difference in case counts (NYT-USAFacts)/USAFacts over time.
+
+The case counts reported in the two sources are similar, and have gotten more similar in the past couple months. During the month of August, NYT consistently reported approximately 0.6% more cases than USAFacts, compared to a difference of 1% throughout May and June.
+
+```{r, fig.height=8, fig.width=10}
+g <- bind_rows(nyt_total, usafacts_total) %>% select(-deaths) %>% spread(source, cases) %>% 
```

Comment:
the legend of this has comparison listed twice and the legend items are numbered, e.g. (USAFacts, 1) -- can we fix this?

Reply:
Plotly didn't seem to want to move the labels to the left, so I did facet_wrap so the labels are above. What do you think?